### ISSUE TITLE
Snapshot creation date in filename or timestamp

### ISSUE THREAD

**Issue by magnusoverli** [danielwelch/hassio-dropbox-sync#8]
Hi!

Great addition to my Hassio set-up!!

I feel like it would make sense to include the date of snapshot creation in the filename or otherwise transfer that to the dropbox backup. Maybe it´s possible to keep original created timestamp when transferring?

**Comment by danielwelch** [comment]
I agree that it would be nice, but I think it would require a significant re-write. Has.io itself determines the file name during the snapshot creation process, so we’d have to rename the file before uploading it. I’ve looked at using the Dropbox python SDK instead of the Dropbox uploaded script, so if the add-on were to move over, this would be a nice additional feature.

**Comment by Smiggel** [comment]
+1 for me

**Comment by lhoracek** [comment]
+1

**Comment by marima84** [comment]
+1 please
